src/d/d_timer.cpp

```diff
@@ -121,14 +126,21 @@ void dDlst_TimerScrnDraw_c::setRupee(short) {
     /* Nonmatching */
 }
 
-/* 8023CEF0-8023CF48       .text dTm_parentPaneScale__FP18fopMsgM_pane_classP18fopMsgM_pane_classf */
+/* 8023CEF0-8023CF48       .text dTm_parentPaneScale__FP18fopMsgM_pane_classP18fopMsgM_pane_classf
+ */
 void dTm_parentPaneScale(fopMsgM_pane_class*, fopMsgM_pane_class*, float) {
     /* Nonmatching */
 }
 
-/* 8023CF48-8023CF98       .text setPaneInitialPos__21dDlst_TimerScrnDraw_cFP18fopMsgM_pane_classff */
-void dDlst_TimerScrnDraw_c::setPaneInitialPos(fopMsgM_pane_class*, float, float) {
+/* 8023CF48-8023CF98       .text setPaneInitialPos__21dDlst_TimerScrnDraw_cFP18fopMsgM_pane_classff
+ */
+void dDlst_TimerScrnDraw_c::setPaneInitialPos(fopMsgM_pane_class* param_1, float param_2,
+                                              float param_3) {
     /* Nonmatching */
+    (param_1->mPosCenterOrig).x = param_2 + (0.5 * (param_1->mSizeOrig).x);
+    (param_1->mPosCenterOrig).y = param_3 + (param_1->mSizeOrig).y * 0.5;
+    fopMsgM_paneTrans(param_1, 0.0, 0.0);
```

Comment:
Heads up, it's probably not going to match unless you include the f suffix on floats -- e.g. `param_2 + (0.5f * param_1->mSizeOrig.x)`. You should include the f suffix on basically every float.